spec/controllers/my/files_controller_spec.rb

```diff
@@ -21,19 +21,24 @@
     end
   end
 
+  after(:each) do
+    cleanup_jetty
+  end
+
   describe "Logged in user" do
-      it "should respond with success and shows the correct documents" do
-        get :index
-        expect(response).to be_successful
-        document_list_ids = assigns[:document_list].map(&:id)
-        # shows documents shared with me
-        expect(document_list_ids).to include(@read_shared_with_me.id)
-        expect(document_list_ids).to include(@edit_shared_with_me.id)
-        # does show normal files
-        expect(document_list_ids).to include(@my_file.id)
-        # doesn't show files shared with other users
-        expect(document_list_ids).to_not include(@unshared_file.id)
-      end
+    it "should respond with success and shows the correct documents" do
+      get :index
+      expect(response).to be_successful
+
+      document_list_ids = assigns[:response].docs.map(&:id)
```

Comment:
Mostly just style changes here...I did change this variable defination however. 

@document_list variable is not used anywhere in the view, everything is driven off the @response.docs . So let's test the thing that actually matters?